data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -39,23 +39,23 @@ spec:
             - /bin/bash
             - -ce
             - |
-              echo 'Installing bridge and tuning CNIs'
-              cni_mount_dir=/opt/cni/bin
-              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
-              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
-              cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
+              if [ find ${cni_mount_dir}/bridge &>/dev/null ] && [ ! test -L ${cni_mount_dir}/bridge ]; then
+                ln -s ${cni_mount_dir}/bridge ${cni_mount_dir}/cnv-bridge
+                ln -s ${cni_mount_dir}/tuning ${cni_mount_dir}/cnv-tuning
+              else
+                cni_mount_dir=/opt/cni/bin
+                sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
```

Comment:
Does not this need to be above the first if?

Reply:
why? it is used only below this line, and the if (of line 42 checks only `${cni_mount_dir}/bridge`
please correct me if you want something else

Reply:
You are right
`cni_mount_dir=/opt/cni/bin` should be before the if, (or just use hardcoded `/opt/cni/bin`)
Thanks